editor.planx.uk/src/lib/graphql.ts

```diff
@@ -45,14 +48,40 @@ const publicHttpLink = createHttpLink({
   headers: { "x-hasura-role": "public" },
 });
 
-const errorLink = onError(({ graphQLErrors }) => {
+const handlePermissionErrors = (message: string, operation: Operation) => {
+  const permissionErrors = [
```

Comment:
There are a number of issues on the Hasura [`graphql-engine`](https://github.com/hasura/graphql-engine) repo requesting better error messages - please see explanation here for context https://github.com/hasura/graphql-engine/issues/6716#issuecomment-1354346094